upsertGitHubTag/deployment/index.js

```diff
@@ -25,6 +26,10 @@ const verifyGitHub = (req, payload) => {
   );
 };
 
+function getProtocol(url) {
+  return url.protocol === "https" ? https : http;
```

Comment:
Inexplicably, per my the results of my experiment in node right now, the url module includes the trailing colon in the protocol, for example `https:`.  So, right now, the comparison is never true, and it always returns the `http` protocol.  Related thought, not sure it matters, but it might be safer to force `http` rather than the other way around, ala:
```
return url.protocol === "http:" ? http : https;
```

Reply:
Good catch, I'd run into this myself in another PR and failed to notice it here: https://github.com/dockstore/lambda/pull/157/files#diff-e8c71a1d88ea008ea7072c2ec67aeb28960108f130c7b2c7dcb75b7a9e6efd7bR9